code/game/area/areas.dm

```diff
@@ -32,7 +32,8 @@
 	var/clockwork_warp_allowed = TRUE // Can servants warp into this area from Reebe?
 	var/clockwork_warp_fail = "The structure there is too dense for warping to pierce. (This is normal in high-security areas.)"
 
-	var/fire = FALSE
+	var/actual_fire = FALSE //If true, theres an actual fire detected in this area
```

Comment:
Why do you have a separate variable for this

Reply:
`fire` is when a fire alarm is triggered, `actual_fire` is when there is a god damn fire in the area

Reply:
i wanna seperate between them, one is by someone activating it when there is no fire and one is triggered when theres a real fire

Reply:
but you know this is actually a bit silly because i only use that for destroy() in firealarm, i guess ill remove that then